.github/workflows/build_test_linux_php8-1.yml

```diff
@@ -46,6 +47,8 @@ jobs:
         #  run: |
         #    sudo apt-get install valgrind
 
+        - name: Setup LCOV
+          uses: hrishikesh-kadam/setup-lcov@v1
```

Comment:
why using hrishikesh-kadam? Is it ok to use? any license requirement?

Reply:
removed